javascript.html.markdown

```diff
@@ -603,31 +603,35 @@ const pi = 3.14;
 
 pi = 4.13; // You cannot do this.
 
-// There is a new syntax for functions in ES6 known as "lambda syntax".
+// There is a new syntax for functions in ES6 known as "arrow function"
+// (lambda syntax in many others languages).
 // This allows functions to be defined in a lexical scope like with variables
 // defined by const and let. 
+// By default, a single statement defined will be the return statement, hence the `return` keyword is not required. 
```

Comment:
Not exactly true:
`() => 3` will return 3, but 
`() => {3}` will not.

The rule here is: if (a) there's only one statement in the function body and (b) you drop the curly braces you can (1) no longer use the return statement and (2!) the resulting value of the statement is the resulting value of the function.